src/models/assetGroupSighting/useDeleteAssetGroupSighting.js

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import axios from 'axios';
+import { get } from 'lodash-es';
+import { formatError } from '../../utils/formatters';
+
+export default function useDeleteAssetGroupSighting() {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+  const [success, setSuccess] = useState(false);
+
+  const deleteAssetGroupSighting = async assetGroupSightingId => {
```

Comment:
gah, i hate to throw this back again but we really need to be using react-query for all our new api hooks

Reply:
Do you want the old delete sighting to be updated to react-query as well?

Reply:
that would be nice, but not a requirement